packages/opentelemetry-api/src/metrics/Metric.ts

```diff
@@ -51,12 +49,6 @@ export interface MetricOptions {
    */
   disabled?: boolean;
 
-  /**
-   * (Measure only, default true) Asserts that this metric will only accept
-   * non-negative values (e.g. disk usage).
-   */
-  absolute?: boolean;
```

Comment:
ValueRecorder was the only thing using `absolute` that I could find. Via https://github.com/open-telemetry/opentelemetry-js/issues/1241 it seems like that should not be configurable. As such, seemed natural to rip out while in here.